dev/setup-envs.sh

```diff
@@ -5,7 +5,7 @@ cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/../
 SHELL_BIN=$(basename "$SHELL")
 
 case "$SHELL_BIN" in
-  *"bash"*) CONFIG_FILE="$HOME/.bashrc"
+  *"bash"*) CONFIG_FILE="$HOME/.bash_profile"
   ;;
   *"zsh"*) CONFIG_FILE="$HOME/.zshrc"
```

Comment:
Should we also change this then?

```suggestion
  *"zsh"*) CONFIG_FILE="$HOME/.zshenv"
```

Reply:
Here is a good ressource to choose which file is the best for `zsh`: https://unix.stackexchange.com/questions/71253/what-should-shouldnt-go-in-zshenv-zshrc-zlogin-zprofile-zlogout